db/models.py

```diff
@@ -50,3 +60,78 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(null=True, blank=True)
```

Comment:
The `created_at` field in the `Order` model should have `auto_now_add=True` to automatically set the field during creation. This change is necessary to meet the task requirements.